src/libNode/FinalBlockProcessing.cpp

```diff
@@ -362,6 +362,8 @@ void Node::ScheduleMicroBlockConsensus() {
 
   auto main_func = [this]() mutable -> void { RunConsensusOnMicroBlock(); };
 
+  SetState(MICROBLOCK_CONSENSUS_PREP);
```

Comment:
Already we set the state in RunConsensusOnMicrkock at start. Infact AFAIK this redundant SetState was removed in prev Pr